apps/meteor/server/routes/avatar/middlewares/auth.js

```diff
@@ -1,22 +0,0 @@
-import { userCanAccessAvatar, renderSVGLetters } from '../utils';
```

Comment:
If this change is not critical to the feat, i'd recommend moving this ts conversion to another PR